packages/quick-edit-extension/src/cfs.ts

```diff
@@ -4,7 +4,7 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-import workersTypes from "raw:@cloudflare/workers-types/experimental/index.d.ts";
+import workersTypes from "raw:workers-types";
```

Comment:
for my understanding, what's the diff between the two?

Reply:
is this possible because we're now resolving the path to `node_modules/@cloudflare/workers-types/experimental/index.d.ts`?

Reply:
Yeah—we now resolve the path in the esbuild plugin rather than from the import statement